can/interfaces/pcan/pcan.py

```diff
@@ -86,9 +86,15 @@ def __init__(self, channel='PCAN_USBBUS1', state=BusState.ACTIVE, bitrate=500000
             Bitrate of channel in bit/s.
             Default is 500 kbit/s.
 
+        :param bool user_bit:
+            If true use the user defined bitrate with SJW, BRP, TSEG2, TSEG1
+            Default is False
         """
         self.channel_info = channel
-        pcan_bitrate = pcan_bitrate_objs.get(bitrate, PCAN_BAUD_500K)
+        if not user_bit:
```

Comment:
Couldn't we solve this without another parameter as well?

```python
try:
    pcan_bitrate = pcan_bitrate_objs[bitrate]
except AttributeError:
    pcan_bitrate = TPCANHandle(bitrate)
```

Reply:
This does remove that default parameter from the access to `pcan_bitrate_objs` but it's wasn't really helping anyways since the `__init__()` has already set it to a default value.

Reply:
Yes your are right, I just thought about legacy issues if someone entered an incorrect baudrate, his program will fail (when he expected the 500k)

Reply:
Yes your are right, I just thought about legacy issues if someone entered an incorrect baudrate, his program will fail (when he expected the 500k)

Reply:
Well, in the long run I guess we will catch more wrong usages than break old programs, so I would change it to the new behaviour.

Reply:
It should be KeyError right?

Reply:
Yep, sorry